### ISSUE TITLE
Reproducing the results from readme file

### ISSUE THREAD

**Issue by brasanko** [dessa-public/DeepFake-Detection#10]
Hi there, how can I reproduce the results seen that can be seen in the readme file? Is it possible to reproduce those results (photos + the corresponding labels about whether the photo is classified as fake or not) within python or one has to use Atlas? Also, is it possible to insert and test custom photos or only those provided by you?

**Comment by brasanko** [comment]
Hi there, how can I reproduce the results seen that can be seen in the readme file? Is it possible to reproduce those results (photos + the corresponding labels about whether the photo is classified as fake or not) within python or one has to use Atlas? Also, is it possible to insert and test custom photos or only those provided by you?

**Comment by Rayhane-mamah** [comment]
Hello,

if by "reproducing the results" you mean train the best model we achieved, then yes you can do it in simple python. You can find the hparams in the [UI](http://deepfake-detection.dessa.com/projects/DeepFake-Detection-Debunked/job_listing). When you have the parameters, you can either retrain the model on your data or download the pre-trained model if you wish.
If however you want to reproduce all 140 experiments that we did, while you could do that manually in python, we find that atlas makes that easier by scheduling all of the 140 experiments in a single [script](https://github.com/dessa-oss/DeepFake-Detection/blob/master/hparams_search.py). 

For your question about custom data, you can put whatever data you wish in the [correct path](https://github.com/dessa-oss/DeepFake-Detection/blob/master/restructure_data.sh#L81) and the pipeline with train a model on that data. Keep in mind that the repo doesn't provide inference code, it only provides training code, so you may have to write that on your own.

Hope this answers your questions?

**Comment by CoinCheung** [comment]
Where could we download the pre-trained model for evaluation please?
